web/src/utils/initStoryblok.test.ts

```diff
@@ -0,0 +1,67 @@
+import { storyblokInit } from "@storyblok/react";
+
+import { logger } from "@/logger";
+
+import { initStoryblok } from "./initStoryblok";
+
+jest.mock("@/config", () => ({
+	publicRuntimeConfig: {
+		storyblok: {
+			ocelotEndpoint: "testEndpoint",
+			accessToken: "testApiKey",
+		},
+	},
+	serverRuntimeConfig: {
```

Comment:
Prob don't need to mock serverRuntimeConfig in this way as we're doing everything in the public runtime